queue/ipfs_cluster.go

```diff
@@ -87,6 +87,9 @@ func (qm *Manager) processIPFSClusterPin(ctx context.Context, d amqp.Delivery, w
 		d.Ack(false)
 		return
 	}
+	if err != nil && err.Error() == "json: Unmarshal(nil) (200)" {
+		qm.l.Infow("compat bug", "error", err)
```

Comment:
Change this to an error log not an info log, it will be more likely to be picked up by manual inspection, or filter out issues with log scrapers.